src/Facilitator.ts

```diff
@@ -70,18 +80,38 @@ export default class Facilitator {
    *
    * @return <any> Object containing chain based subscriptionQueries.
    */
-  public static getSubscriptionDetails(): any {
+  public static getSubscriptionDetails(): SubscriptionInfo {
     return {
       origin: {
-        subGraphEndPoint: 'ws://localhost:8000/subgraphs/name/openst/ost-composer',
+        wsSubGraphEndPoint: 'ws://localhost:8000/subgraphs/name/openst/ost-composer',
+        httpSubGraphEndPoint: 'http://localhost:8000/subgraphs/name/openst/ost-composer',
         subscriptionQueries: {
-          stakeRequested: 'subscription{stakeRequesteds{id amount'
-          + ' beneficiary gasLimit gasPrice gateway nonce staker stakeRequestHash}}',
+          stakeRequested: 'subscription{stakeRequesteds(orderDirection: desc, first: 1){id}}',
         },
       },
       auxiliary: {
-        subGraphEndPoint: 'ws://localhost:8000/subgraphs/name/openst/ost-composer',
-        subscriptionQueries: { stakeRequested: 'subscription{stakeRequesteds{id}}' },
+        wsSubGraphEndPoint: 'ws://localhost:8000/subgraphs/name/openst/ost-composer',
+        httpSubGraphEndPoint: 'http://localhost:8000/subgraphs/name/openst/ost-composer',
+        subscriptionQueries: {
+          stakeRequested: 'subscription{stakeRequesteds(orderDirection: desc, first: 1){id}}',
+        },
+      },
+      fetchQueries: {
+        stakeRequested: 'query ($uts: BigInt!) {\n'
+        + '  stakeRequesteds(where: {uts_gt: $uts}, orderDirection: asc, limit: 100) {\n'
```

Comment:
In an architecture where we have one facilitator working for only one gateway pair, we should be good with not adding `contractAddress` to these queries.

Reply:
Agreed 👍